client/state/selectors/is-rtl.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import { getCurrentUser } from 'state/current-user/selectors';
+
+/**
+ * Returns the isRtl value for the current user.
```

Comment:
Since there's no `isRtl` property for the `currentUser`, I'd change this to:

`Whether the current user is in right-to-left mode.`

or something along these lines.